src/js/tooltip.js

```diff
@@ -371,12 +384,22 @@ class Tooltip {
 	 * @returns {Object} An object with values `left`, `right`, `top` and `bottom`
```

Comment:
This JSDOC comment looks to be for the old version `calculateTooltipRect`. We need to add JSDOC comments for the modified/new methods